### PR TITLE
Improve hover and active background-color for header tool buttons in …

### LINKED ISSUE #2123: In Panel>headerItems, button styling on hover should be more transparent.
This issue exist on my local instance of toolbox in Light Theme:
http://localhost:3000/app/panels/sizing

and in deployed develop in Light Theme:
https://toolbox-dev.xh.io/app/panels/sizing

Here is a screenshot of the button in my local instance when hovered over:
image

The issue does NOT exist in Dark Theme.

I'm seeing the same bug in a client app running the latest HR commits on my dev machine.

### PR DESCRIPTION
**Pull request by cnrudd**
…light theme. (fixes #2123)

Hoist P/R Checklist
-------------------

**Pull request authors:** Review and check off the below. Items that do not apply can also be
checked off to indicate they have been considered. If unclear if a step is relevant, please leave
unchecked and note in comments.

- [x] Caught up with `develop` branch as of last change.
- [x] Added CHANGELOG entry.
- [x] Reviewed for breaking changes: no breaking changes.
- [x] Updated doc comments:  not required.
- [x] Reviewed and tested on Mobile:  not required.
- [x] Created Toolbox branch: not required.

**If your change is still a WIP**, please use the "Create draft pull request" option in the split
button below to indicate it is not ready yet for a final review.

> **Pull request reviewers:** when merging this P/R, please consider using a *squash commit* to
> collapse multiple intermediate commits into a single commit representing the overall feature
> change. This helps keep the commit log clean and easy to scan across releases. PRs containing a
> single commit should be *rebased* when possible.

### CODE REVIEW

File: desktop/cmp/button/Button.scss [hunk 1]
```diff
@@ -370,3 +370,22 @@
...
+//------------------------
+// Header "tool" buttons in panels and docks
+// Adjustment needed only in light theme.
+//------------------------
+*:not(.xh-dark),
+*:not(.bp3-dark) {
+  .xh-dock-view__header,
+  .xh-panel-header__items {
+    .xh-button.bp3-button.xh-button--minimal {
+      &.xh-button--intent-none {
```
Comment (cnrudd) [inline desktop/cmp/button/Button.scss:383]: I followed the suggestion from PanelHeader.scss, where we don't worry about other intents or styles other than minimal when header buttons are concerned.

**Review by amcclain — COMMENTED** [review]
This seems very closely related to #2127 which I filed only yesterday - can you take a look at folding in a fix for that here as well?

File: desktop/cmp/button/Button.scss [hunk 2]
```diff
@@ -370,3 +370,22 @@
     }
   }
 }
+
+//------------------------
+// Header "tool" buttons in panels and docks
+// Adjustment needed only in light theme.
+//------------------------
+*:not(.xh-dark),
+*:not(.bp3-dark) {
```
Comment (amcclain) [inline desktop/cmp/button/Button.scss:379]: Do you know if we need the `:not(.bp3-dark)` here?  Is there a case where that would be applied but `.xh-dark` is not?

**Comment by cnrudd** [comment]
> This seems very closely related to #2127 which I filed only yesterday - can you take a look at folding in a fix for that here as well?

@amcclain #2127 fixed in latest commit.

File: desktop/cmp/button/Button.scss (hunk 2 above)
Comment (cnrudd) [reply desktop/cmp/button/Button.scss]: @amcclain I've removed `:not(.bp3-dark)`.  I had just thrown it in because it seemed to be a pattern up higher.

**Review by amcclain — APPROVED** [review]
Looks good, thanks for addressing these issues.

**Pull request merged**